README.dev.md

```diff
@@ -92,6 +95,17 @@ virtualenv -p python .venv
 .venv\Scripts\activate.ps1
 python -m pip install -r requirements.txt
 ```
+
+#### Conda virtual environment
```

Comment:
I think Conda instructions would have to be duplicated in the Linux and Mac sections as well.